src/Security/Security/help/Get-AzSqlVulnerabilityAssessmentBaseline.md

```diff
@@ -0,0 +1,194 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Security.dll-Help.xml
+Module Name: Az.Security
+online version:
```

Comment:
Please add online version, its format should be https://docs.microsoft.com/powershell/module/{ModuleName}/{CmdletName}, e.g. https://docs.microsoft.com/powershell/module/az.security/get-azsqlvulnerabilityassessmentbaseline

Reply:
pending